terraform/cyhy_reporter_ec2.tf

```diff
@@ -34,7 +34,7 @@ resource "aws_instance" "cyhy_reporter" {
 
   root_block_device {
     volume_type = "gp2"
-    volume_size = "${local.production_workspace ? 200 : 8}"
+    volume_size = "${local.production_workspace ? 100 : 8}"
```

Comment:
We have to discuss if/where we are going to store the reports each week after they are generated and sent out.